conflation/trace_fetching/mapillary.py

```diff
@@ -382,3 +541,16 @@ def get_tile_from_lon_lat(lon: float, lat: float, zoom: int) -> tuple[int, int]:
     xtile = int((lon + 180.0) / 360.0 * n)
     ytile = int((1.0 - math.asinh(math.tan(lat_rad)) / math.pi) / 2.0 * n)
     return xtile, ytile
+
+
+def get_lon_lat_from_tile(zoom: int, x: int, y: int) -> tuple[float, float]:
+    """
+    Turns a Slippy map tile at a given zoom into a lon/lat measurement.
+
+    TODO: What if the tile is "out of bounds"?
```

Comment:
Question here -- if we happen to be at the largest (x, y) tile of a zoom, does this function still work? I'm assuming not. E.g. if we call this with (1, 1, 1)

Reply:
why would we ever pass invalid tile coords to this function? it wouldnt be because we iterate over tiles that are out of bounds? basically the user passes in a bbox of interest, we turn that into tile ranges. if that bbox of interest intersected the anti-meridian we need to split it at the anti meridian and do 2 boxes. is this the case that you are talking about, we will need special handling to split the box. a good test case is fiji :wink:

Reply:
I was actually referring to when we calculate the z14 tiles, where we do the +1, +1 on the x, y to create 4 different z14 tiles. But I just realized that we still won't have invalid coords, since the z14 tiles we calculate should always map to a "square" in z5.

Good call on the anti-meridian edge case. I wonder if somehow the current logic already works since I take the min(x) and min(y) to iterate over at z5. It's extremely inefficient but there's a chance it might already work. On another note, how would you even pass in a bbox that intersects the anti-meridian (e.g. only covering Fiji)? Doesn't the fact that we require min_lon,max_lon means you have to go around the world?